pkgs/build-support/setup-hooks/auto-patchelf.py

```diff
@@ -264,10 +262,16 @@ def auto_patchelf(
     missing = [dep for dep in dependencies if not dep.found]
 
     # Print a summary of the missing dependencies at the end
+    print(f"auto-patchelf: {len(missing)} dependencies not be satisfied")
```

Comment:
nit: not be -> are not

Reply:
oops, mean to write "could not be"

Reply:
Fixed in https://github.com/NixOS/nixpkgs/pull/167397/commits/a7fc2f6392be849f811c0a3f29f559311dc695df